src/Commands/MeltorCommand.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+namespace Meltor\Commands;
+
+use Exception;
+use Illuminate\Console\Command;
+use Illuminate\Database\ConnectionInterface;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\DB;
+use Meltor\Meltor;
+
+class MeltorCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'meltor:generate
+                {--f|force               : Execute without confirmation. }
+                {--i|ignoreProblems      : Leave out structures that cannot be processed. }
+                {--s|separateForeignKeys : Put foreign keys at the end of the file. Avoids problems with constraint checks. }
+                {--t|testrun             : Perform a db structure comparison testrun (will need to drop and restore your DB). }
+                {--r|restore             : Restore the db backup from a previous run, stops after that. }
```

Comment:
Might be clearer to have this as a separate command. Changing the whole behavior and purpose by adding a switch is a bit strange.